ledger-service/jwt/src/main/scala/com/digitalasset/jwt/JwtVerifier.scala

```diff
@@ -43,31 +43,74 @@ object JwtVerifier {
   }
 }
 
+final case class LeewayOptions(
+    leeway: Option[Long] = None,
+    expiresAt: Option[Long] = None,
+    issuedAt: Option[Long] = None,
+    notBefore: Option[Long] = None,
```

Comment:
I am wondering if `leeway` and 3 others are conflicting between each other?

Reply:
The rest (if they are present) override the `leeway` parameter.